examples/onnxruntime/resnet50/resnet50.py

```diff
@@ -227,6 +264,7 @@ def main():
     if flags.QPS:
         print("resnet50, Rate = {} QPS ".format(
             format((((flags.batch) / (sum(latency) / len(latency)))), '.2f')))
```

Comment:
Same as comment for inceptionv3, remove the cold start run from the average

Reply:
Done